### PR TITLE
solution

### PR DESCRIPTION
**Pull request by igor-kashchenko**

### CODE REVIEW

**Review by stadnytskyi-ivan — APPROVED** [review]
Good job)

File: src/makeCalculator.js [hunk 1]
```diff
@@ -37,7 +37,41 @@
...
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
```
Comment (alexandra-protyanova) [inline src/makeCalculator.js:55]: is not the best name for variables

File: src/makeCalculator.js [hunk 2]
```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```
Comment (alexandra-protyanova) [inline src/makeCalculator.js:40]:
```suggestion
  const calculator = {
```

File: src/makeCalculator.js [hunk 3]
```diff
@@ -37,7 +37,41 @@
...
+      return a / b;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, n) {
```
Comment (alexandra-protyanova) [inline src/makeCalculator.js:65]:
```suggestion
    operate(operation, value) {
```

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**